content/en/docs/tasks/security/health-check/index.md

```diff
@@ -110,14 +105,10 @@ continuously failed health checks.
 
 ## Cleanup
 
-*   To disable health checking on Citadel:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
-    {{< /text >}}
+*   To disable health checking on Citadel: follow the [uninstall instructions](/docs/setup/install/kubernetes/#Uninstall) to remove.
```

Comment:
this looks different than the original one, before it said install and now uninstall

Reply:
I think it should be a bug before. this section is `cleanup` ...